hail/src/main/scala/is/hail/services/DeployConfig.scala

```diff
@@ -0,0 +1,89 @@
+package is.hail.services
+
+import java.io.{File, FileInputStream}
+
+import is.hail.utils._
+import org.json4s.{DefaultFormats, Formats, JValue}
+import org.json4s.jackson.JsonMethods
+
+object DeployConfig {
+  lazy val get: DeployConfig = fromConfigFile()
+
+  def fromConfigFile(file0: String = null): DeployConfig = {
+    var file = file0
+
+    if (file == null)
+      file = System.getenv("HAIL_DEPLOY_CONFIG_FILE")
+
+    if (file == null) {
+      val fromHome = s"${ System.getenv("HOME") }/.hail/deploy-config.json"
+      if (new File(fromHome).exists())
+        file = fromHome
+    }
```

Comment:
I added a check for `/deploy-config/deploy-config.json` to the python-side of things after one-to-many-times forgetting to set the environment variable. We should at least be consistent and add it here or remove it there, IMO.

Reply:
Oops, good catch, I had meant to just copy the Python functions but messed this one up.